web/src/message_scroll.js

```diff
@@ -14,10 +14,23 @@ import * as unread_ui from "./unread_ui";
 let hide_scroll_to_bottom_timer;
 export function hide_scroll_to_bottom() {
     const $show_scroll_to_bottom_button = $("#scroll-to-bottom-button-container");
+    $("#scroll-to-bottom-button-clickable-area").off("mouseenter");
+
     if (message_viewport.bottom_message_visible() || message_lists.current.visibly_empty()) {
         // If last message is visible, just hide the
         // scroll to bottom button.
         $show_scroll_to_bottom_button.removeClass("show");
+
+        // "Scroll Button" tooltip needs to be explicitly destroyed because the [data-reference-hidden] property does not work accurately under animation. Track the issue here - "https://github.com/atomiks/tippyjs/issues/1154".
+        $("#scroll-to-bottom-button-clickable-area").on("mouseenter", () => {
+            setTimeout(() => {
+                const tippyInstance = $("#scroll-to-bottom-button-clickable-area")[0]._tippy;
+                if (tippyInstance) {
+                    tippyInstance.destroy();
+                }
+            }, 500);
```

Comment:
why 500ms timer?

Reply:
When I remove the timer , the tippy instance destroyed is not the actual instance which I want to.

Might be possible that Tippy.js might need a brief moment to fully initialize the tooltip instance, and if you attempt to destroy it immediately, the instance might not be ready or available. By introducing a small delay using setTimeout, you give Tippy.js some time to complete its initialization process, ensuring that the tooltip instance is available for destruction.